docs/modules/item.md

```diff
@@ -35,7 +35,20 @@ and it would involve removing a current enchantment, add the flag `-o`. If you a
 enchantment level onto an item which is not normally allowed (often referred to as "unsafe"), add the flag `-u`, though
 this requires an extra permission (`nucleus.enchant.unsafe`).
 
+## Repair
+
+Nucleus can be used to repair items in a player's inventory using `/repair`. By default, `/repair` will only target the item held in a player's main hand. By using and combining the availible flag arguments, specific inventory locations or the entire inventory can be repaired.
```

Comment:
I didn't realise the `en_US` spelling was "availible" ;)

(it's "available")

Reply:
I never claimed to be good at spelling.